scale-tests/kafka_cassandra_streaming_test.py

```diff
@@ -175,58 +202,70 @@ def append_submission(output_file: str, dispatcher_app_name: str, submission_id:
 
     with open(args["<infrastructure_file>"]) as f:
         infrastructure = json.loads(f.read())
-        # We might make this script handle multiple Zookeeper/Kafka/Cassandra
-        # clusters in the future. For now it only handles a single cluster of
-        # each service.
-        zookeeper = infrastructure['zookeeper'][0]
-        kafka = infrastructure['kafka'][0]
+        zookeepers = infrastructure['zookeeper']
```

Comment:
ZooKeepers are not used, so we could drop reading them from the infrastructure file.

Reply:
cca684cc7011a7210374d3b3cc6085e315be4ad1